shells/configuration/whitelisted.js

```diff
@@ -28,3 +28,4 @@ import '../lib/database/firebase-upload.js';
 // TODO(sjmiles): TensorFlowJs (tfjs, also part of ml5) uses `new Function()` which requires `unsafe-eval` csp
 import '../services/textclassifier-service.js';
 import '../lib/services/tf.js';
+import '../lib/services/random.js';
```

Comment:
Should reference the file in build/services. Same with the other two for that matter. See next comment.

Reply:
done, will fix others in follow on PR.

Reply:
done